man/udevadm.xml

```diff
@@ -395,6 +395,26 @@
             then each matching result is ORed, that is, all children of each specified device are triggered.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--initialized-match</option></term>
+          <listitem>
+            <para>Trigger events for devices that are already initialized by
+            <command>systemd-udevd</command>, and skip devices that are not initialized yet.</para>
```

Comment:
maybe explain what initialized means, i.e. something like this:

> Initialized devices are those for which at least one udev rule already completed execution – for any action but "remove" — that set a property or other device setting (and thus has an entry in the udev device database). Devices are no longer considered initialized if a "remove" action is seen for them (which removes their entry in the udev device database). Note that devices that have no udev rules are never considered initialized, but might still be announced via the sd-device API (or similar). Typically, it is thus essential that applications which intend to use such a match, make sure a suitable udev rule is installed that sets at least one property on devices that shall be matched.

Or something like that.